R/model-modules.R

```diff
@@ -551,7 +553,7 @@ enw_missing <- function(formula = ~1, data) {
     # Get report dates that cover all reference dates up to the max delay
     rep_w_complete_ref <- enw_reps_with_complete_refs(
       data$new_confirm[[1]],
-      max_delay = data$max_delay[[1]],
```

Comment:
Max delay is not a list anymore, so I don't think we need the indexing? It is now just
`max_delay = data$max_delay`
Or do you think this is not safe?

Reply:
I was thinking because the input is always a data.table of row one and therefore I thought you had to index it but you are right this is working and it doesn't seem unsafe.